GUIDELINES.md

```diff
@@ -171,6 +171,18 @@ Example:
 }
 ```
 
+### User Permissions
+
+#### Fetching objects
+
+When fetching objects if the User has permissions to fetch it but he lacks a specific property permission then that property will not be returned in the response.
```

Comment:
We discussed the possibility to simple return `null` instead of making the properties `optional` in the response. If we would make them `null` that would mean that the integrator wouldn't know if the property doesn't have value or if the User don't have the right to access it. Making it optional will make it explicit that the User don't have access to it but will make it less friendly for our integrators because they will have to have some if conditions for those properties.